### PR TITLE
Add Forward PID Subsystem to the WheelSystem and inverted a motor in the intake system

### PR DESCRIPTION
**Pull request by jwong101**

### CODE REVIEW

**Comment by jwong101** [comment]
This closes #78 #77

**Comment by LoadingPleaseWait** [comment]
Can you add unit tests for calculating speed? This pull request is also for #77.

**Comment by jwong101** [comment]
The unit test in the AccelerometerSystem mainly checks the velocity calculations. Also, there were several bugs in the lever calculation for the intake system:
1. The limit switch did not stop the lever if it was given input aside from the timed shoot.
2. There was a slight delay between getting the input for the timed shoot and actually starting the execution of the shoot.
3. Even after the input for moving the lever backwards interrupted the timed shoot, the timed shoot execution continued immediately after the command was given for moving the lever backwards.
4. No catch for when multiple input buttons are pressed at the same time for the lever.

The bugs above were fixed and test cases were added in the IntakeSystemTest to test them.
Also, since the lever moves slightly back for each timed shoot, the time for moving the lever backwards during the time shoot was decreased.

**Comment by LoadingPleaseWait** [comment]
Can you make it so that the manual input for lever does not use the limit switch?

**Comment by LoadingPleaseWait** [comment]
The variables, `XAccel`, `YAccel`, `ZAccel` and `Vinitial` in `AccelerometerTest` are not named in lower camel case. Also, the method `getRawValues` in `AccelerometerSystem` should be renamed since it starts with get but has side effects.

File: src/org/impact2585/frc2016/systems/AccelerometerSystem.java [hunk 1]
```diff
@@ -45,7 +45,16 @@
...
+	private long prevTime;
+	private double deltaTime;
+	
+	private double prevXVelocity;
+	private double prevYVelocity;
+	private double prevZVelocity;
+	
+	private double XVelocity;
+	private double YVelocity;
+	private double ZVelocity;
```
Comment (LoadingPleaseWait) [inline src/org/impact2585/frc2016/systems/AccelerometerSystem.java]: These variables also need to be named in lower camel case.

**Comment by LoadingPleaseWait** [comment]
It looks like the new timing for the lever might not work. Unit test that logic. Also, rename `I2Cbus`, `FIFObuffer` and `FIFOcount` so that they are in lower camel case.

**Pull request merged**